homework-submission/main.js

```diff
@@ -0,0 +1,67 @@
+const List = props => {
+  return (
+    <li>
+      {props.description},{props.deadline}
+    </li>
+  );
+};
+
+class TodoList extends React.Component {
+  staticList = [
+    { description: 'Get out of bed', deadline: 'Wed Sep 13 2017' },
+    { description: 'Brush teeth', deadline: 'Thu Sep 14 2017' },
+    { description: 'Eat breakfast', deadline: 'Fri Sep 15 2017' },
+  ];
+
+  dynamicList = [
+    {
+      id: 1,
+      description: 'Get out of bed',
+      deadline: '2017-09-11',
+      done: true,
+    },
+    {
+      id: 2,
+      description: 'Brush teeth',
+      deadline: '2017-09-10',
+      done: false,
+    },
+    {
+      id: 3,
+      description: 'Eat breakfast',
+      deadline: '2017-09-09',
+      done: false,
+    },
+  ];
+
+  render() {
+    return (
+      <div>
+        <h1>static List</h1>
+        <List description={this.staticList[0].description} deadline={this.staticList[0].deadline} />
+        <List description={this.staticList[1].description} deadline={this.staticList[1].deadline} />
+        <List description={this.staticList[2].description} deadline={this.staticList[2].deadline} />
+
+        <div>
+          <h1>Dynamic List</h1>
+          {this.dynamicList.map(todoList => (
+            <div>
+              <li
```

Comment:
It's very important to note having an `li` inside of a `div`.

As you can see there is no point of having an `li` inside of a `div`, but the browser even doesn't know how to deal with that situation.

![Screen Shot 2019-07-04 at 03 20 26](https://user-images.githubusercontent.com/25272822/60633386-b8df1500-9e0a-11e9-919a-0952c8040042.png)